cmd/workspace.go

```diff
@@ -80,6 +80,14 @@ var (
 		RunE:  workspaceUserAdd,
 	}
 
+	workspaceUserUpdateCmd = &cobra.Command{
+		Use:   "update user role",
+		Short: "Update a user to a workspace",
+		Long:  "Update a user to a workspace",
```

Comment:
Maybe "Update a user's role for a workspace"

Reply:
fixed.